ci/tests/test_geneformer/test_geneformer_model.py

```diff
@@ -40,9 +50,115 @@ def test_ensure_data_validity_raising_error_with_missing_ensembl_id_column(self)
                                 (['SAMD11', 'PLEKHN1', 'HES4'], False),
                              ]
     )
-    def test_ensembl_data_is_caught(self, gene_symbols, raises_error):
-        self.data.var['gene_symbols'] = gene_symbols
+    def test_ensembl_data_is_caught(self, geneformer, mock_data, gene_symbols, raises_error):
+        mock_data.var['gene_symbols'] = gene_symbols
         if raises_error:
             with pytest.raises(ValueError):
-                self.geneformer.process_data(self.data, "gene_symbols")
+                geneformer.process_data(mock_data, "gene_symbols")
+        else:
+            geneformer.process_data(mock_data, "gene_symbols")
+    def test_get_embs_cell_mode(self, geneformer, mock_data):
```

Comment:
the new geneformer is larger than the previous one so I can understand why it takes longer. 1min is no biggie for me but the more models we add, the bigger it will become